Shapes/shapes/v2/language_map.ttl

```diff
@@ -0,0 +1,140 @@
+###############################################################################
+# RML Language Map and language shortcut shape                                #
+# Copyright Dylan Van Assche, IDLab - UGent - imec (2023)                     #
+###############################################################################
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix schema: <http://schema.org/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rml: <http://w3id.org/rml/core/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> . 
+
+schema:RMLLanguageMapShape
+    a sh:NodeShape ;
+    sh:ignoredProperties (rdf:type) ;
+    sh:name "rml:languageMap" ;
+    sh:description """
+    Represents a Language Map.
+    """ ;
+    sh:message """
+    rml:LanguageMap must specify an rml:template or rml:constant with the IRI 
+    of the language.
+    """ ;
+
+    # Exactly one rml:template or one rml:constant is required
+    sh:property [
```

Comment:
that repeats again, I just mention to further support my comment that it might be better to give IRIs to the shapes and reuse them instead of redefining them again and again.